src/Core/Varargs/PrintfFormatParser.cs

```diff
@@ -18,16 +18,13 @@
  */
 #endregion
```

Comment:
Do you mean `MsPrintfFormatParser` inherited from ` PrintfFormatParser` with empty impementation part should be created?